cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/KafkaCluster.java

```diff
@@ -110,6 +110,10 @@ public static String headlessName(String cluster) {
         return cluster + KafkaCluster.HEADLESS_NAME_SUFFIX;
     }
 
+    public static String kafkaPodName(String cluster, int podId) {
```

Comment:
Why is this a static method? It can be non-static and we would not need to pass the cluster name. In the places where you use this you anyway have an object of this type to get the cluster name.

Reply:
My first intention was to have methods like static `kafkaClusterName` but you are right where we need pod and/or pvc we have a cluster instance with all the needed information.